order-management-service/api/src/main/java/com/microservices/ordermanagement/api/dto/AddDetailDto.java

```diff
@@ -15,6 +16,6 @@
 
     private Integer orderId; // if exists
 
-    @NotNull
-    private Integer userId;
+    @NotBlank
+    private String username;
 }
```

Comment:
no userId?

Reply:
Yes, I think user can be identified by login everywhere